src/main/scala/com/raphtory/core/model/graph/visitor/Edge.scala

```diff
@@ -7,6 +7,12 @@ trait Edge extends EntityVisitor {
   def src():Long
   def dst():Long
   def explode():List[ExplodedEdge]
+  def weightOrHistory(weight : String="weight") : Float = {
```

Comment:
I think this is great to have Naomi! For the naming, we need to differentiate between spatial and temporal features so just having `weight` is misleading because (1) it's generally understood as a spatial attribute and (2) as one single value. My suggestion is to go for something like `temporalTotalWeight` with not mention to history and defaulting to `1.0` when no property is specified.

Reply:
We are aggregating the different temporal edges here, so maybe `totalWeight` or `aggregateWeight` if we want to be more explicit? I.e., what we are computing ends up being a spatial attribute as we reduce over the temporal dimension...

Reply:
I have changed this to totalWeight now, thanks!